### ISSUE TITLE
Items are disappearing when they're dropped/broken

### ISSUE THREAD

**Issue by Noctleigh** [ResqDiver1317/AC-Sauce-1_10_2_modpack#28]
Issue: Items are disappearing when they're dropped/broken

Troubleshooting: 
Initial issue was reported with Quantum Tank (Quantum Storage) being placed on Thickened Glass (Extra Utilities)
 - Placed Quantum Tank on glass and broke: Behaves normally
 - Placed Quantum Tank filled with water and and SemiSweet broke it with pickaxe: Behaves normally
 - Placed Quantum Tank filled with Creosote Oil and SemiSweet broke it with pickaxe: Behaves normally
 - Started to log incident, and issue happened again. It is intermittent.
 - Collected her Tinkers Pickaxe (Manyllun with Enchants) to try myself. When I dropped it, it disappeared.
 - Made note that none of these happened due to ClearLag. That was timed separately.

Suspected Cause: Either chunk or Thickened glass that things were being dropped onto

Reported on: 10/27

Reported by: Munch

**Comment by ResqDiver1317** [comment]
I just tested this with about 50 breaks of the tank with various tools, on my alabaster floor and on thickened glass and was unable to get it to disappear at all. 

I did check their github and saw there was a few mentions of similar issues reported. We are 2 versions behind on that mod and although their github does not specifically say if either of the 2 newer versions actually address and resolve this, it's worth a shot. In the morning, I am going to start working through our modlist and see what all needs updated and try to get the modpack update built and pushed to the server/clients.
